builds/install/misc/firebird.conf

```diff
@@ -244,11 +244,17 @@
 # ----------------------------
 # Number of cached database pages
 #
-# This sets the number of pages from any one database that can be held
-# in cache at once. If you increase this value, the engine will
-# allocate more pages to the cache for every database. By default, the
-# SuperServer allocates 2048 pages for each database and the classic
-# allocates 256 pages per client connection per database.
+# This setings is used to cache in memory database pages. 
+# It specify maximum number of pages which can be cached.
+# In SuperServer it is per database, in Classic per client connection.
+# Per-database configurable. This value can be overrided by databases.conf or 
+# directly in database file by issuing command:
+# gfix -b[uffers] BUFFERS
+# Priority of setings is: direcly in database, databases.conf, firebird.conf.
```

Comment:
settings

Reply:
I see here one typo `direcly` should be `directly`